### PR TITLE
watcher: Do not initialize the config in watcher

### PR DESCRIPTION
**Pull request by theopolis**
This should fix an error in the watchdog, where an exception is thrown when attempting to use the database. Previously we used the `Config`, and on initialization it constructed a `Schedule`, which attempted to look up the last-failed query from the database.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [1944](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1944/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [5061](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5061/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [1185](https://jenkins.osquery.io/job/osqueryPullRequestOSX/1185/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [1945](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1945/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [5062](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5062/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit d72ef3dc15814f7da6f946f74d0c18aeeefea133 (Job results: [1186](https://jenkins.osquery.io/job/osqueryPullRequestOSX/1186/)) failed one or more tests (**macOS/OS X**).

File: osquery/main/windows/daemon.cpp [hunk 1]
```diff
@@ -16,12 +16,9 @@
 
 #include <osquery/core.h>
 #include <osquery/flags.h>
-#include <osquery/logger.h>
```
Comment (theopolis) [inline osquery/main/windows/daemon.cpp]: @poppyseedplehzr, this is the line that should be included, right?

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [1946](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1946/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [5063](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5063/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [1947](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1947/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [1187](https://jenkins.osquery.io/job/osqueryPullRequestOSX/1187/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [5064](https://jenkins.osquery.io/job/osqueryPullRequestLinux/5064/)) failed one or more tests (**Linux**).

**Comment by osqueryer** [comment]
:-1: The commit a19c1d8de94d2832b339c11240881bd4fec096dc (Job results: [1188](https://jenkins.osquery.io/job/osqueryPullRequestOSX/1188/)) failed one or more tests (**macOS/OS X**).

**Comment by osqueryer** [comment]
:-1: The commit 2ec17ac3737cef96b0b0122c2162fc43b7f2cbad (Job results: [1948](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1948/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 2ec17ac3737cef96b0b0122c2162fc43b7f2cbad (Job results: [1949](https://jenkins.osquery.io/job/osqueryPullRequestWindows/1949/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit 2ec17ac3737cef96b0b0122c2162fc43b7f2cbad (Job results: [85](https://jenkins.osquery.io/job/osqueryPullRequestLinux/85/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit 93f6f365d286c832ec754683ff9b22cdc1382b44 (Job results: [90](https://jenkins.osquery.io/job/osqueryPullRequestLinux/90/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit afe0eb694c78d24545ee026e9a112d8248d5fbe1 (Job results: [91](https://jenkins.osquery.io/job/osqueryPullRequestLinux/91/)) failed one or more tests (**FreeBSD**).

**Comment by osqueryer** [comment]
:-1: The commit afe0eb694c78d24545ee026e9a112d8248d5fbe1 (Job results: [92](https://jenkins.osquery.io/job/osqueryPullRequestLinux/92/)) failed one or more tests (**FreeBSD**).

**Pull request merged**